src/Microsoft.OData.Core/Microsoft.OData.Core.cs

```diff
@@ -7,8 +7,7 @@
 // </copyright>

```

Comment:
These changes are intentional.

I had to add a new error message string to the string resources. To do it, I added it to the .txt file, and then ran the T4 template to regenerate the corresponding .cs files.